### PR TITLE
[PT-261] Metrics wrapper

### PR DESCRIPTION
**Pull request by brennervaz**

### CODE REVIEW

File: src/lib/utils/constants.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+export const INITIAL_CONFIG = {
+  context: {
+    type: 'metric',
+  },
+}
+
+export const METRIC_TYPES = {
```
Comment (brennervaz) [inline src/lib/utils/constants.js:10]: @rcmachado not sure what types of metrics we would be covering here

File: src/lib/utils/constants.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+export const INITIAL_CONFIG = {
+  context: {
+    type: 'metric',
```
Comment (brennervaz) [inline src/lib/utils/constants.js:4]: @rcmachado I'm adding `{ type: 'metric' }` to the global context so we can filter/match those in log ingestion

File: src/lib/metrics/index.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    for(const METRIC_TYPE of METRIC_TYPES) {
+      this.createLogger(METRIC_TYPE)
+    }
+  }
+
+  createLogger(type) {
+    const { context: configContext } = getGlobalConfig()
+    const context = {
+      ...configContext,
+      metric_type: type,
```
Comment (brennervaz) [inline src/lib/metrics/index.js]: @rcmachado every log will have `{ metric_type: 'typeofmetrichere' }` in the context

File: src/lib/utils/constants.js (hunk 1 above)
Comment (rcmachado) [reply src/lib/utils/constants.js:10]: Count for now is enough.

**Review by rcmachado — APPROVED** [review]
Looks good so far

**Pull request merged**